app/assets/stylesheets/navbar_left.scss

```diff
@@ -135,4 +112,78 @@
     }
     /* ---- end override app/stylesheets/vendor/autoSuggest.css ---- */
   }
+
+  .info-bar {
+    margin-top: 25px;
+
+    .section {
+      margin-bottom: 20px;
+
+      > .title {
+        border-bottom: 1px solid $border-grey;
+        padding-bottom: 5px;
+
+        h5 {
+          color: $text-dark-grey;
+          font-size: $font-size-base;
+          margin: 0;
+        }
+      }
+
+      .content {
+        color: $text-grey;
+        font-size: $font-size-small;
+        line-height: 18px;
+        padding: 10px 0;
+
+        p,
+        ul {
+          margin: 0;
+        }
+
+        ul {
+          margin-bottom: 5px;
+          padding-left: 0;
+          li { list-style: none; }
+        }
+
+        .btn-link {
+          font-size: $font-size-small;
+          padding-left: 0;
+        }
+
+        > #invite_code {
+          box-sizing: border-box;
+          font-size: $font-size-small;
+          height: 30px;
+          width: 100%;
+        }
+
+        > .right-service-icons {
+          padding: 10px {
+            bottom: 0;
+          }
+          text-align: center;
+
+          .social_media_logos-facebook-24x24,
```

Comment:
Selector `social_media_logos-facebook-24x24` should be written in lowercase with hyphens